concepts/system-model/index.md

```diff
@@ -49,6 +47,46 @@ The principal message primitives of the networking communication layer are:
 Transaction calls execute against local state and are not broadcast to the network, per standard Ethereum.
 :::
 
+### P2P networking protocols
+P2P networking protocols are Ethereum devp2p, RLPx as transport to an ethereum wire protocol (eth66). Modifications for the Autonity Protocol consensus and communication model are made:
+
+- the wire protocol (eth66) is modified to add messages for Tendermint BFT consensus rounds
+- peer to peer gossiping is separated into two channels, ethwire and consensus, separating _transaction_ and _consensus_ level gossiping on to different TCP ports (defaults of `30303` and `20203` respectively).
+
+### Separate channels for transaction and consensus gossiping
+The segregation of transaction and consensus traffic into different channels on prevents the risk of high transaction traffic interfering with consensus traffic and delaying the consensus process. Further, gossiping separation improves network scalability and robustness as consensus is shielded from transaction volume growth, whilst the separation of concerns allows each gossiping channel to be managed independently (e.g. socket buffering). For example, a validator node operator can setup their IP and port configuration to prioritise a robust network for the consensus channel.
+
+Autonity implements [Tendermint BFT consensus](/concepts/consensus/pos/) as an independent consensus protocol running alongside the ethereum wire protocol. Logically, this can be considered as a separate Autonity Consensus Network (ACN) layer. The consensus protocol leverages node information obtained through the wire protocol's node discovery mechanisms. If the consensus channel TCP port default (`20203`) is not used, then the custom `ip:port` information is specified as part of the enode url. If the enode URL provides consensus channel information, this is used to connect to the validator node at the specified IP:port combination. Otherwise, the consensus protocol assumes that the consensus channel IP is the same as the transaction channel IP, and the consensus channel port is fixed at the default port number `20203`.
```

Comment:
OK, I have deleted:

>If the enode URL provides consensus channel information, this is used to connect to the validator node at the specified IP:port combination. Otherwise, the consensus protocol assumes that the consensus channel IP is the same as the transaction channel IP, and the consensus channel port is fixed at the default port number `20203`.

This paragraph now reads:

>Autonity implements [Tendermint BFT consensus](/concepts/consensus/pos/) as an independent consensus protocol running alongside the ethereum wire protocol. Logically, this can be considered as a separate Autonity Consensus Network (ACN) layer. By default, the consensus protocol is configured to use port `20203` and assumes the IP to be the same as the Ethereum wire protocol IP. If a validator wishes to use a different IP and/or port combination for the consensus network, they can specify this information in the enode URL.

Commit bd8e50b56ec2e7f357a86d01c7be9cb50d2c481e

Reply:
Resolving - discussed with @vpiyush who is happy with edit.